gsmphys/radiation_gases.f

```diff
@@ -402,15 +403,17 @@ subroutine gas_init                                               &
 
                 do j = 1, JMXCO2
                   do i = 1, IMXCO2
-                    co2vmr_sav(i,j,imo) = co2dat(i,j) * 1.0e-6
+                    co2vmr_sav(i,j,imo) = co2dat(i,j) * 1.0e-6          &
+     &                                  * co2_scaling
                   enddo
                 enddo
               enddo
 
               if ( me == 0 ) then
                 print *,' - Using co2 monthly 2-d data from user',      &
      &                ' provided data set:',co2usr_file
-                print *, iyr,cline(1:94),co2g1,'  GROWTH RATE =', co2g2
+                print *, iyr,cline(1:94),co2g1 * co2_scaling,           &
+     &                  '  GROWTH RATE =', co2g2
```

Comment:
Do you want to add a note that you are rescaling the data in that file by co2_scaling?